src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTip.xaml

```diff
@@ -0,0 +1,63 @@
+﻿<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:macos="http://uno.ui/macos"
+	mc:Ignorable="d macos">
+
+	<SolidColorBrush x:Key="ToolTipForeground" Color="{StaticResource SystemBaseHighColor}" />
+	<SolidColorBrush x:Key="ToolTipBackground" Color="{StaticResource SystemChromeMediumLowColor}" />
+	<SolidColorBrush x:Key="ToolTipBorderBrush" Color="#000000" />
+	<Thickness x:Key="ToolTipBorderThemePadding">8,5,8,7</Thickness>
+
+	<!-- Default style for Windows.UI.Xaml.Controls.ToolTip -->
+	<Style TargetType="ToolTip">
+		<Setter Property="Foreground" Value="{ThemeResource ToolTipForeground}" />
+		<Setter Property="Background" Value="{ThemeResource ToolTipBackground}" />
+		<Setter Property="BorderBrush" Value="{ThemeResource ToolTipBorderBrush}" />
+		<Setter Property="BorderThickness" Value="{ThemeResource ToolTipBorderThemeThickness}" />
+		<!--<Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />-->
+		<Setter Property="FontSize" Value="12" />
+		<Setter Property="Padding" Value="{ThemeResource ToolTipBorderThemePadding}" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="ToolTip">
+					<ContentPresenter x:Name="LayoutRoot"
+						BorderThickness="{TemplateBinding BorderThickness}"
+						Background="{TemplateBinding Background}"
+						BackgroundSizing="OuterBorderEdge"
+						BorderBrush="{TemplateBinding BorderBrush}"
+						MaxWidth="320"
+						Content="{TemplateBinding Content}"
+						ContentTransitions="{TemplateBinding ContentTransitions}"
+						ContentTemplate="{TemplateBinding ContentTemplate}"
+						Padding="{TemplateBinding Padding}"
+						TextWrapping="Wrap">
+
+						<!--<VisualStateManager.VisualStateGroups>
```

Comment:
Add a comment explaining why this is omitted. Also, the convention [elsewhere](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml) is to use the `win:` prefix for parts of the style that aren't yet supported in Uno.